helm/charts/hpe-csi-driver/README.md

```diff
@@ -65,13 +50,11 @@ helm repo update
 
 Install the latest chart:
 ```
-# Helm 3
 helm install hpe-csi hpe/hpe-csi-driver --namespace kube-system -f myvalues.yaml
-
-# Helm 2, for Nimble, Install with HPENodeInfos CRD's enabled explicitly
-helm install --name hpe-csi hpe/hpe-csi-driver --namespace kube-system -f myvalues.yaml --set crd.nodeInfo.create=true
 ```
 
+**Note**: values.yaml is optional if no parameters are overridden from defaults.
+
 ### Upgrading the Chart
 
 To upgrade the chart, specify the version you want to upgrade to as below. Please do NOT re-use a full blown `values.yaml` from prior versions to upgrade to later versions. Always use `values.yaml` from corresponding release from [values.yaml](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver)
```

Comment:
Can we fix the below chunk to look consistent:

List the available version of the plugin:
```
helm repo update
helm search repo hpe-csi-driver -l
```

Select the target version to upgrade as below:
```
helm upgrade hpe-csi hpe/hpe-csi-driver --namespace kube-system --version=x.x.x.x -f myvalues.yaml
```

Reply:
done..thanks.